src/meta/raft-store/src/sm_v002/leveled_store/leveled_map.rs

```diff
@@ -89,76 +230,59 @@ impl LeveledMap {
     pub(crate) fn replace_frozen_levels(&mut self, b: StaticLeveledMap) {
         self.frozen = b;
     }
+
+    pub(crate) fn leveled_ref_mut(&mut self) -> LeveledRefMut {
+        LeveledRefMut::new(&mut self.writable, &self.frozen)
+    }
+
+    pub(crate) fn leveled_ref(&self) -> LeveledRef {
+        LeveledRef::new(Some(&self.writable), &self.frozen)
+    }
 }
 
 #[async_trait::async_trait]
 impl<K> MapApiRO<K> for LeveledMap
 where
-    K: Ord + fmt::Debug + Send + Sync + Unpin + 'static,
+    K: MapKey + fmt::Debug,
```

Comment:
Why not force MapKey Debuggable?

Reply:
Because to `MapApi`, the key does not have to be `Debug`?